Documentation/tutorial/docfx_getting_started.md

```diff
@@ -41,7 +41,31 @@ docfx docfx_project\docfx.json --serve
 ```
 
 Now you can view the generated website on http://localhost:8080.
-3. Use *DocFX* with a Build Server
+
+3. Use *DocFX* integrated with Visual Studio
+---------------
+
+*Step1.* Install the [`docfx.console`](https://www.nuget.org/packages/docfx.console/) (formerly `docfx.msbuild`) nuget package on the project that you want to document.
+It add itself to the build targets and add the `docfx.json` configuration file along with other files.
+
+*Step2.* Compile, a `_site` folder is generated with the documentation.
+
+> [!NOTE]
+> *Possible warning*:
+> - *Cache is corrupted*: if your project targets multiple frameworks, you have to indicate one to be the main for the documentation, through the [`TargetFramework` property](https://github.com/dotnet/docfx/issues/1254#issuecomment-294080535) in `docfx.json`:
```

Comment:
*Cache is corrupted* should be a false warning, a bug to fix 😄 I will update the document when the bug is fixed..